src/redux/home-page-slides/home-page-slides.actions.js

```diff
@@ -0,0 +1,104 @@
+import {
+  ADD_SLIDE,
+  ADD_SLIDE_TO_STORE,
+  DELETE_SLIDE,
+  GET_AVAILABLE_SLIDES,
+  GET_SLIDE,
+  GET_SLIDES,
+  REMOVE_SLIDE_FROM_STORE,
+  SET_AVAILABLE_SLIDES,
+  SET_SLIDE,
+  SET_SLIDE_ERROR,
+  SET_SLIDE_LOADING,
+  SET_SLIDES,
+  SET_SLIDES_CURRENT_PAGE,
+  SET_SLIDES_DRUG_AND_DROP_LIST,
+  SET_SLIDES_PAGES_COUNT,
+  SET_SLIDES_PER_PAGE,
+  UPDATE_SLIDE, UPDATE_SLIDES_ORDER
+} from './home-page-slides.types';
+
+export const setSlides = (payload) => ({
+  type: SET_SLIDES,
+  payload
+});
+
+export const setSlidesCurrentPage = (payload) => ({
+  type: SET_SLIDES_CURRENT_PAGE,
+  payload
+});
+
+export const setSlidesPerPage = (payload) => ({
+  type: SET_SLIDES_PER_PAGE,
+  payload
+});
+
+export const setSlidesPagesCount = (payload) => ({
+  type: SET_SLIDES_PAGES_COUNT,
+  payload
+});
+
+export const getSlides = (payload) => ({
+  type: GET_SLIDES,
+  payload
+});
+
+export const setSlideLoading = (payload) => ({
+  type: SET_SLIDE_LOADING,
+  payload
+});
+
+export const setSlideError = (payload) => ({
+  type: SET_SLIDE_ERROR,
+  payload
+});
+
+export const setSlidesDrugAndDropList = (payload) => ({
+  type: SET_SLIDES_DRUG_AND_DROP_LIST,
```

Comment:
Try to remain your type names relatively short, rename this type to: `SET_DRUG_AND_DROP_SLIDES`

Reply:
Fixed